server/runtime/src/main/java/org/infinispan/server/configuration/security/LdapUserPasswordMapperConfiguration.java

```diff
@@ -22,11 +22,13 @@ static AttributeSet attributeDefinitionSet() {
    }
 
    void build(LdapSecurityRealmBuilder ldapRealmBuilder) {
-      LdapSecurityRealmBuilder.UserPasswordCredentialLoaderBuilder builder = ldapRealmBuilder.userPasswordCredentialLoader();
-      builder.setUserPasswordAttribute(attributes.attribute(FROM).get());
-      if (!attributes.attribute(VERIFIABLE).get()) {
-         builder.disableVerification();
+      if (attributes.attribute(FROM).get() != null) {
```

Comment:
I think the better option here is to add the usercredentialbuilder by default if `direct-evidence-verification` is `false` in `LdapRealmConfiguration`.